src/python/WMCore/ACDC/CouchFileset.py

```diff
@@ -99,9 +99,22 @@ def add(self, files, mask = None):
         _add_
 
         Add files to this fileset
+
+        Note: if job was lumi based splitted, then we do not have
+        reliable events information. If job was event based splitted,
+        then we do not have reliable lumi information.
         """
         filteredFiles = []
         if mask:
+            for f in files:
+                # There is no LastEvent for last job of a file
+                if mask['LastEvent'] and mask['FirstEvent']:
+                    f['events'] = mask['LastEvent'] - mask['FirstEvent']
```

Comment:
We don't care uploading the last event to couch since we have already firstevent and # of events.
In this example lastevent was not in couch, but was used to build the adcd docs, the only case it's not available is for the last job of the workflow (or perhaps last job of the fileset)... I still think we don't need +1 here :-)